test/default_meta/tests.jl

```diff
@@ -0,0 +1,13 @@
+using Documenter
+using Test
+
+const pages = [
+    "Home" => "index.md",
+]
+
+makedocs(
+    sitename = "Test", pages = pages, doctest = true,
+    default_meta = Dict(:DocTestSetup => :(x = 42))
```

Comment:
Just to say, one could also allow (instead or in addition) named tuples:
```suggestion
    default_meta = (DocTestSetup = :(x = 42), )
```
OK that's a bit awkward in the single argument list. But it is a tad closer to how the content of the `@meta` blocks look (no `:` in front of names, `=` instead of `=>`)

Anyway I am totally fine either way, just wanted to bring it up

Reply:
NamedTuples would be nice...